.eslintrc.yml

```diff
@@ -55,6 +58,8 @@ rules:
     - error
   react-native/no-color-literals:
     - error
+  '@typescript-eslint/no-unused-vars': 
+    - 2
```

Comment:
why were these changes to `np-unused-vars` and `@typescript-eslint/no-unused-vars` added and why are they different?

Reply:
the normal `no-unused-vars` rule must be disabled because it conflicts with the newly added one from typescript.

the typescript one is needed over the normal one because otherwise eslint will complain about unused variables in type definitions of functions

```ts
type NotificationHandler = (arg: Notifications.Notification) => void;
type ResponseHandler = (arg: Notifications.NotificationResponse) => void;
```